tests/test_study.py

```diff
@@ -528,3 +532,28 @@ def test_load_study(storage_mode, cache_mode):
         # Test loading an existing study.
         loaded_study = optuna.study.load_study(study_name=study_name, storage=storage)
         assert created_study.study_id == loaded_study.study_id
+
+
+@pytest.mark.parametrize('storage_mode', STORAGE_MODES)
+def test_running_study(storage_mode):
```

Comment:
Please update the function name since `RunningStudy` was renamed to `InTrialStudy`.

Reply:
Fixed: https://github.com/pfnet/optuna/pull/380/commits/aaa33ca9f90976286d8095b95eae0dc4dffe56c3. Thanks!